apps/ledger-live-mobile/src/screens/BleDevicePairingFlow/BleDeviceScanning.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { FlatList } from "react-native";
+import { Flex, InfiniteLoader, Text } from "@ledgerhq/native-ui";
+import { BluetoothMedium } from "@ledgerhq/native-ui/assets/icons";
+import { BleErrorCode } from "react-native-ble-plx";
+import { useBleDevicesScanning } from "@ledgerhq/live-common/ble/hooks/useBleDevicesScanning";
+import { useNavigation } from "@react-navigation/native";
+import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { ScannedDevice } from "@ledgerhq/live-common/ble/types";
+import { getDeviceModel } from "@ledgerhq/devices";
+import { DeviceModelId } from "@ledgerhq/types-devices";
+
+import { knownDevicesSelector } from "../../reducers/ble";
+import TransportBLE from "../../react-native-hw-transport-ble";
+import LocationRequired from "../LocationRequired";
+import BleDeviceItem from "./BleDeviceItem";
+import type { BleDevicePairingFlowProps } from "./index";
+
+const BluetoothThingy = () => (
```

Comment:
not sure if "Thingy" is the best name 😂
For the borderRadius, doesn't "50%" work?

Reply:
Ahah I did not name this, but will change it for `BluetoothLogo`

For the borderRadius, i'm going to try and let you know ;)

Reply:
50% gives a square 🟦 - and i cannot put 100% or 99%, i get the error: "JSON value '100%' of type NSString cannot be converted to NSNumber". I think that's why @thomasrogerlux put `9999px`. 

Note: this will be replaced by an animated BLE logo in the future. But we don't have this animation yet. So i think it's good

Reply:
I see. Yeah, no problems with 9999px for now then.